guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -18,6 +18,11 @@ For more details, see https://projects.theforeman.org/issues/37252 and https://p
 
 Foreman 3.10 only supported {EL} 9 as experimental, but with this release it is fully supported.
 
+=== Running Foreman on Debian 12 is supported
```

Comment:
The title is very similar to the title above it about EL 9, so it'd be inconsistent. Though I'd agree it's a bit redundant perhaps.

I'm not a fan of `{Team}` because it expands to `Foreman community`. I don't think we can speak on behalf of the community. These are the official docs and I think the author is already implied.